src/go/k8s/tests/e2e/produce-consume/01-redpanda-cluster.yaml

```diff
@@ -12,10 +12,10 @@ spec:
   resources:
     requests:
       cpu: 1
-      memory: 2Gi
+      memory: 100M
```

Comment:
I needed to do this because tests were flaky locally for me without this. I think 100M is enough